cypress-io/cypress
# fix: Prevent assertion text from osscilating when it has a trailing space

### User facing changelog
Assertions will no longer 'vibrate' in the command log when asserting on the text of elements with trailing whitespace.

### Additional details

This doesn't happen any more:

https://user-images.githubusercontent.com/3003404/189177706-fc084be0-bce6-4c1d-a29e-efcf7477f68b.mov

This does not close the referenced issue; while it was discovered as part of https://github.com/cypress-io/cypress/issues/23699, it's not directly related to the user's report.

### Steps to test
Take the newly added test and run it against develop (removing the `cy.on('command:retry'` block) to see the effect visually. No longer occurs in this branch. 

### How has the user experience changed?

### PR Tasks
- [n/a] Has the original issue (or this PR, if no issue exists) been tagged with a release in ZenHub? (user-facing changes only)
- [n/a] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [n/a] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?
